src/certificate.ts

```diff
@@ -18,17 +18,18 @@ export async function responseCertificate(
   res.setHeader('Content-Type', 'application/pdf');
 
   const doc = new PDFDocument({ size: 'A4' });
+
   doc.pipe(res);
 
   renderFirstPage(doc, talent, course);
 
   doc.addPage();
 
-  renderSecondPage(doc, course.topics);
+  renderSecondPage(doc, course.topics, course.type);
```

Comment:
It's getting more complex if you add more arguments. I would reduce it to a single object argument or even better, only bypass `doc`. 

```suggestion
  renderSecondPage(doc);
```

or 
```suggestion
  renderSecondPage({ 
    doc, 
    topics: course.topics, 
    type: course.type
  });
```

`topics` and `type` are already part of it.
Same for other functions.